### PR TITLE
chore: Migrate CI to Gh actions

### PR DESCRIPTION
**Pull request by sawood14012**
# Description

This Code Adds Github actions as CI 

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by arajkumar — APPROVED** [review]
lgtm

File: requirements.in [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-gunicorn==19.7.1
+boto3==1.7.84
+daiquiri
+flask
+gevent==1.4.0
+numpy
+scipy==1.2.1
+gunicorn
 raven[flask]
+setuptools==41.0.0
```
Comment (rootAvish) [inline requirements.in:9]: Setuptools itself on principle is never installed as part of requirements. This you need to install through `pip install --upgrade pip` separately.

File: requirements.txt [hunk 2]
```diff
@@ -2,27 +2,65 @@
...
+s3transfer==0.1.13
+    # via boto3
+scipy==1.2.1
+    # via -r requirements.in
+six==1.12.0
+    # via python-dateutil
+werkzeug==0.15.3
+    # via flask
+
+# The following packages are considered to be unsafe in a requirements file:
```
Comment (rootAvish) [inline requirements.txt:65]: See pip-compile itself gives you an error :)

**Review by rootAvish — COMMENTED** [review]
Just a minor change, otherwise LGTM

**Pull request merged**